libs/langchain/langchain/document_loaders/concurrent.py

```diff
@@ -75,7 +74,11 @@ def from_filesystem(
             show_progress=show_progress,
         )
         if isinstance(parser, str):
-            blob_parser = get_parser(parser)
+            if parser == "default" and cls.get_parser != GenericLoader.get_parser:
```

Comment:
Would it be better to try/except NotImplementedError instead of checking method != ?

Reply:
Sure that works as well I'll change